src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,21 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+
+        Machine truck1 = new Truck();
+        Machine bulldozer1 = new Bulldozer();
+        Machine excavator1 = new Excavator();
```

Comment:
It is bad practice to use numbers in variable names
```suggestion
        Machine truck = new Truck();
        Machine bulldozer  = new Bulldozer();
        Machine excavator  = new Excavator();
```